packages/provider/src/RelayClient.ts

```diff
@@ -110,6 +113,23 @@ export class RelayClient {
     }
     this.rawConstructorInput = rawConstructorInput
     this.logger = rawConstructorInput.overrideDependencies?.logger ?? console
+    this.wrapEthersJsProvider()
+  }
+
+  wrapEthersJsProvider (): void {
+    const provider = this.rawConstructorInput.provider as any
+    if (typeof provider.getSigner === 'function') {
```

Comment:
Afaik, ethers had a _provider marker property instead of checking for method

Reply:
Yes, but not all `ethers providers` have a `getSigner` and we cannot really use a provider that does not have a signer.